docs/ui/auth_admin.rst

```diff
@@ -0,0 +1,16 @@
+.. _ref_ui_auth_admin:
+
+==========
+Auth Admin
+==========
+
+The auth page inside the EdgeDB UI allows you to set up authentication for
+your app built into the EdgeDB server. The details for setting up auth
+admin are in their own guide :ref:`here <ref_guide_auth>`.
```

Comment:
Hmm, I could go either way but my first impression is that Auth is still more of a guide than just a documented item and it would feel a bit weird to have a page in the Guide section that just links to somewhere else. But then again, bringing it under the UI makes the UI seem that more impressive as a tool and having a section in the UI that just links to somewhere else is also a bit weird!

Although the tie-breaker to prefer the current layout for me is probably that when you click on Auth under Guides there is a nice submenu that shows that Auth has a certain independence (e.g. "Email and password" via a user's own UI) that makes it a little more than just a part of the UI. So it does feel like a semi-autonomous feature that just happens to *usually* be used through our UI. I'll ping @raddevon to see what he thinks too.

Reply:
Related: I just slipped in a tocdepth change for auth from 3 to 1 so that the Overview page just shows the following three instead of all the submenus:

Select your method for detailed configuration:

* Built-in UI
* Email and password
* OAuth

Reply:
Yeah, now that I think about it, we already have a better place for reference-level documentation about the configuration values themselves: the Reference section! Consider this comment withdrawn, but open to more discussion here.

I'll defer to @raddevon about the `tocdepth` thing.